src/App.js

```diff
@@ -1,23 +1,10 @@
-import logo from './logo.svg';
+import Header from './components/UI/header/Header';
 import './App.css';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div>
+      <Header></Header> 
```

Comment:
if there are no children of your component, please use self-closing tags.
`<Header />`